protocol/contracts/trucoV1/Structs.sol

```diff
@@ -21,21 +22,20 @@ library CardsStructs {
         ReTruco,
         ValeCuatro,
         Envido,
-        EnvidoEnvido,
         RealEnvido,
         FaltaEnvido
     }
 
     enum Response {
         Accept,
         DontAccept,
-        EnvidoCount
+        Envido
     }
 
     struct GameState {
         int8 playerTurn; // player index
         int8[] teamPoints; //points indexed by team id (in this case a player is a team)
-        // revealed cards by player
+        bytes1[6] revealed_cards; // 
```

Comment:
I'm still not comfortable with this struct, since played cards should have a pointer to who played it and in which order :thinking: .. Seems to basic to fit for that use case